main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -639,11 +637,12 @@ object Completion {
   /**
    * Represents an auto-import completion.
    *
-   * @param eff           the effect to complete and use.
+   * @param name          the name to complete.
+   * @param qualifiedName the qualified name to use.
    * @param doc           the documentation associated with the effect.
    * @param ap            the anchor position for the use statement.
    */
-  case class AutoUseEffCompletion(eff: Symbol.EffectSym, doc: String, ap: AnchorPosition) extends Completion
+  case class AutoUseCompletion(name: String, qualifiedName: String,  doc: String, ap: AnchorPosition) extends Completion
```

Comment:
I don't think this is a good idea. Its better to keep them separate-- that way we retain more precise information and we can use different icons etc.